nemo_asr/run_eval.py

```diff
@@ -50,6 +50,8 @@ def download_audio_files(batch):
         audio_paths = []
         durations = []
 
+        # import ipdb; ipdb.set_trace()
```

Comment:
remove this

Reply:
Sorry, missed removing my debug prompt, removed it in the updated script